woocommerce/woocommerce
# Clean up `woocommerce-admin` package.json

### All Submissions:






### Changes proposed in this Pull Request:



Closes #32417.

- Removed unnecessary commands and keys in `package.json`
- Re-order package.json keys via [sort-package-json](https://github.com/keithamus/sort-package-json)

### How to test the changes in this Pull Request:

Review the changes and see if we're good to remove them. It might be easier by reviewing the first commit 31b169b1cb14991ac2f508c985ea3b444714e8a5.

### Other information:




### Changelog entry

Dev: Clean up `woocommerce-admin` package.json

### FOR PR REVIEWER ONLY:
